src/verify_ce_config.py

```diff
@@ -8,19 +8,22 @@
 import re
 import sys
 
+
 def error(msg):
     sys.exit("ERROR: " + msg)
 
+
 def warn(msg):
     print "WARNING: " + msg
 
+
 # Verify that the HTCondor Python bindings are in the PYTHONPATH
 try:
     import classad
     import htcondor
 except ImportError:
-    error("Could not load HTCondor Python bindings. " + \
-          "Please ensure that the 'htcondor' and 'classad' are in your PYTHONPATH")
+    error("Could not load HTCondor Python bindings. "
+          + "Please ensure that the 'htcondor' and 'classad' are in your PYTHONPATH")
```

Comment:
```suggestion
          "Please ensure that the 'htcondor' and 'classad' are in your PYTHONPATH")
```
(don't need a + to concatenate two string literals)